### ISSUE TITLE
Collating 

### ISSUE THREAD

**Issue by Phantom11265** [clj-pdf/clj-pdf#171]
I'm collating 16 PDF documents together. However, they all say 1 on each PDF file. It never concatenates after 1. How can I fix this issue? 

![image](https://user-images.githubusercontent.com/49042422/56325037-01e9cc00-612e-11e9-955b-ab7e17bc39e3.png)

**Comment by yogthos** [comment]
Would you be able to create a sample project exhibiting the issue.

**Comment by Phantom11265** [comment]
So I have about 16 PDF files that are all set up the same way. I have "Page " in the footer but that's about it (for page numbering). If I load 1 file that has more than 1 page of data, then it will work just fine (it will say page 1, page 2, page 3, etc.). However, if I collate the files, then they never increment after one. 
![image](https://user-images.githubusercontent.com/49042422/56368757-4ec4b580-61b5-11e9-920a-253b15920c6d.png)

Here's an example of the collated file. 
[merged-reports.pdf](https://github.com/clj-pdf/clj-pdf/files/3094872/merged-reports.pdf)
Hopefully, this helps.

My only other thought would be to go through each individual file and turn off the numbering.

**Comment by yogthos** [comment]
Yeah nothing jumps out at me there, the code for collating is [here](https://github.com/clj-pdf/clj-pdf/blob/764102ebec1b7c331a3c52594f5c2719045f1867/src/clj/clj_pdf/core.clj#L562), and it should just be adding pages from each PDF to a new document. Might be a downstream bug in a [LibrePDF](https://github.com/LibrePDF/OpenPDF). I just bumped up to the latest version for the dependency in the `2.3.4` release.
